readthedocs/projects/models.py

```diff
@@ -144,11 +144,18 @@ class AddonsConfig(TimeStampedModel):
     Everything is enabled by default.
     """
 
-    DOC_DIFF_DEFAULT_ROOT_SELECTOR = "[role=main]"
-    LINKPREVIEWS_DEFAULT_ROOT_SELECTOR = "[role=main] a.internal"
-    LINKPREVIEWS_DOCTOOL_NAME_CHOICES = (
-        ("sphinx", "Sphinx"),
-        ("other", "Other"),
+    DEFAULT_ROOT_SELECTOR = ", ".join(
+        [
+            "[role=main]",
+            "main",
+            "article",  # Pelican, Docusaurus, Antora, Jekyll
+            "div#content",  # AsciiDoctor
+            "main > div > div.md-content",  # Material for MkDocs
```

Comment:
Don't think this rule will ever be applied, since there is a `main` selector before it, similar thing with `article#main`